### DISCUSSION TITLE
Resuming training after stopping

### DISCUSSION THREAD

**Discussion by Sean3216** [ContinualAI/avalanche#1323] [Q&A] [answered]
Warm greetings.

I would like to ask about the possibility of continuing learning after stopping. I know that I can obtain the model and save its parameters after training with certain CL strategies within Avalanche. But, is it possible to continue a training strategy after I save the parameter? or should I start the training from scratch?

Thank you.
Regards

**Comment by AntonioCarta** [comment, marked as answer]
Yes, you can [save checkpoints of the strategy](https://avalanche.continualai.org/how-tos/checkpoints). Keep in mind that this is a beta feature and it is still quite slow in some cases. We will speed it up in the next version.

**Reply by Sean3216** [reply]
Thanks! I'll check it out later and should I have any troubles, I will post it on the discussion again as a different topic.
